docs/github.md

```diff
@@ -65,7 +65,8 @@ able to create a branch for a revision.
 $ # Do your work
 $ jj commit
 $ # Jujutsu automatically creates a branch
-$ jj git push --change $revision
+$ # Push change "mw", letting Jujutsu automaticcaly create a branch called "push-mwmpwkwknuz"
```

Comment:
typo in "automaticcaly" (sorry if that was my typo :))

Reply:
done